app/src/common/AppBar/AppBar.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
```

Comment:
the `common` folder is meant to store reusable components (different kinds of selects for example). I would have put this in the `app` folder since it's part of the layout.